go.mod

```diff
@@ -24,14 +24,93 @@ require (
 	k8s.io/client-go v0.22.2
```

Comment:
since it is capi uplift, I would expect k8s.io/* deps to be updated to `v0.23.0-alpha.4` https://github.com/kubernetes-sigs/cluster-api/blob/main/go.mod#L31-L35

Reply:
Done. I have chosen it to be v0.23.0